docs/guides/integration/github.md

```diff
@@ -67,37 +67,6 @@ jobs:
 
 This will respect the Python version pinned in the project.
 
-Or, when using a matrix, as in:
-
-```yaml title="example.yml"
-strategy:
-  matrix:
-    python-version:
-      - "3.10"
-      - "3.11"
-      - "3.12"
-```
-
-Provide the version to the `python install` invocation:
-
-```yaml title="example.yml" hl_lines="14 15"
-name: Example
-
-jobs:
-  uv-example:
-    name: python
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Install uv
-        uses: astral-sh/setup-uv@v4
-
-      - name: Set up Python ${{ matrix.python-version }}
-        run: uv python install ${{ matrix.python-version }}
```

Comment:
IIUC https://github.com/astral-sh/uv/issues/9376#issuecomment-2501200380, this method would be inneffective in case a `.python-version` file exists.